### PR TITLE
Added new release notes

### PR DESCRIPTION
**Pull request by rvroon**

### CODE REVIEW

File: docs/products/access/release-notes/index.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
...
+## Release date: 2023-01-24
+
+### Features
+* The end-users is now redirected to the `post-logout-url` after a OIDC logout request, even when the provided ID token is recently expired or there is no active session for that client.
+
+### Bugs 
+* The [Device API](../api-reference/end-user/device-v4.md) now returns a timestamp for `lastLogin`.
+* The `removeScopes` feature in the [Customize Access Token Web Hook](../topics/integration-extension/hooks/customize-access-token/customize-access-token-hook.md) now also works when used together with [Custom registration](../topics/custom-registration/index.md).
+
+## Release date: 2023-01-24
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:16]: This is doubled...?

File: docs/products/access/release-notes/index.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
 
 In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [OneWelcome Support](https://support.onewelcome.com).
 
+## Release date: 2023-01-24
+
+### Features
+* The end-users is now redirected to the `post-logout-url` after a OIDC logout request, even when the provided ID token is recently expired or there is no active session for that client.
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:10]: maybe just `end-user`, plural sounds weird

Comment (jmpavlec) [reply docs/products/access/release-notes/index.md:10]:
```suggestion
* The end-user is now redirected to a whitelisted `post-logout-url` after a OIDC logout request, even when the provided ID token is recently expired or there is no active session for that client.
```

File: docs/products/access/release-notes/index.md [hunk 3]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
...
+
+### Bugs 
+* The [Device API](../api-reference/end-user/device-v4.md) now returns a timestamp for `lastLogin`.
+* The `removeScopes` feature in the [Customize Access Token Web Hook](../topics/integration-extension/hooks/customize-access-token/customize-access-token-hook.md) now also works when used together with [Custom registration](../topics/custom-registration/index.md).
+
+## Release date: 2023-01-24
+
+### Improvements
+* For [Custom registration events](../appendix/access-events.md#custom-registration-audit-events) we now also log the `Client id`, so it is easier to correlate the events to an client or mobile device.
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:19]:
```suggestion
* For [Custom registration events](../appendix/access-events.md#custom-registration-audit-events), we now log the `Client id` to make it easier to correlate the events to an client or mobile device.
```

File: docs/products/access/release-notes/index.md [hunk 4]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
...
+
+### Bugs
+* While adding a mobile authentication type "SMS" the `SMS sender id` always returned a validation error in the UI. This is now fixed.
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:22]:
```suggestion
* While adding a mobile authentication type "SMS", the `SMS sender id` always returned a validation error in the UI. This is now fixed.
```

File: docs/products/access/release-notes/index.md [hunk 5]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
...
 In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [OneWelcome Support](https://support.onewelcome.com).
 
+## Release date: 2023-01-24
+
+### Features
+* The end-users is now redirected to the `post-logout-url` after a OIDC logout request, even when the provided ID token is recently expired or there is no active session for that client.
+
+### Bugs 
+* The [Device API](../api-reference/end-user/device-v4.md) now returns a timestamp for `lastLogin`.
+* The `removeScopes` feature in the [Customize Access Token Web Hook](../topics/integration-extension/hooks/customize-access-token/customize-access-token-hook.md) now also works when used together with [Custom registration](../topics/custom-registration/index.md).
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:14]:
```suggestion
* The `removeScopes` feature in the [Customize Access Token Web Hook](../topics/integration-extension/hooks/customize-access-token/customize-access-token-hook.md) will now be respected when used together with [Custom registration](../topics/custom-registration/index.md).
```

File: docs/products/access/release-notes/index.md (hunk 1 above)
Comment (rvroon) [reply docs/products/access/release-notes/index.md]: Fixed the date

File: docs/products/access/release-notes/index.md [hunk 6]
```diff
@@ -4,6 +4,28 @@ This section contains release notes for OneWelcome Access.
...
+* The end-user is now redirected to a whitelisted `post-logout-url` after a OIDC logout request, even when the provided ID token is recently expired or there is no active session for that client.
+
+### Bugs 
+* The [Device API](../api-reference/end-user/device-v4.md) now returns a timestamp for `lastLogin`.
+* The `removeScopes` feature in the [Customize Access Token Web Hook](../topics/integration-extension/hooks/customize-access-token/customize-access-token-hook.md) will now be respected when used together with [Custom registration](../topics/custom-registration/index.md).
+
+## Release date: 2023-01-18
+
+### Improvements
+* For [Custom registration events](../appendix/access-events.md#custom-registration-audit-events), we now log the `Client id` to make it easier to correlate the events to an client or mobile device.
```
Comment (jmpavlec) [inline docs/products/access/release-notes/index.md:19]:
```suggestion
* For [Custom registration events](../appendix/access-events.md#custom-registration-audit-events), we now log the `Client id` to make it easier to correlate the events to a client or mobile device.
```

**Review by jmpavlec — APPROVED** [review]

**Pull request merged**